src/App.scss

```diff
@@ -21,16 +21,16 @@ iframe {
   &__header {
     margin-bottom: 1em;
   }
-}
+} */
 
-.UserInfo {
+/* .UserInfo {
   font-weight: bold;
-}
+} */
 
-.CommentList {
+/* .CommentList {
```

Comment:
The CSS styling for the .CommentList component has been commented out which might affect the display of the comments.